cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -3,247 +3,378 @@
    Bureau structure page at /the-bureau/bureau-structure/.
    ========================================================================== */
 
+@margin__em: unit(@grid_gutter-width / @base-font-size-px, em);
+@spine-width__px: 5px;
+@spine-color: @gray-40;
+@node-width__desktop__px: 405px;
+@node-width__tablet__px: 300px;
+@node-width__mobile__px: none;
+@node-border-width__px: 3px;
+@node-border-color: @gray-20;
+
 .org-chart {
+    padding: @margin__em;
+    border: 1px solid @node-border-color;
+    margin-top: @margin__em;
+    background: @gray-10;
+
+    .respond-to-max(@bp-xs-max, {
+        padding: 0;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/dca185a0d8c0205efa146b0695e88fa92ea3df97